browser.js

```diff
@@ -1,27 +1,27 @@
 require.config({
   scriptType: 'text/javascript;version=1.8',
-  baseUrl: 'node_modules/',
```

Comment:
I would prefer to keep `baseUrl` mapped to `node_modules` & instead just configure `browser` and `shims` paths, I think adding following to `paths` would do: 

```js
paths: {
  'browser': '../src/browser',
  'shims': '../src/shims',
  //...
}
```

Also how about calling directory `src` instead of `js`, just because most other projects tend to do that.